Dockerfile.int

```
@@ -0,0 +1,11 @@
+FROM python:3.7
+ENV PYTHONUNBUFFERED 1
+ENV DJANGO_SETTINGS_MODULE uwpath_backend.int
+RUN mkdir /code
+WORKDIR /code
+COPY requirements.txt /code/
+RUN pip install psycopg2-binary==2.8.6
+RUN pip install -r requirements.txt
+COPY . /code/
+ENV PATH /code/:$PATH
+EXPOSE 8000
```

Change this to 
```suggestion
EXPOSE $PORT
```
Since GCP will provide the port number